src/ConstructiveSolidGeometry/PointsAndVectors/Points.jl

```diff
@@ -47,7 +47,33 @@ function CartesianPoint{T}(;
     CartesianPoint{T}(T(x),T(y),T(z))
 end
 
-zero(PT::Type{<:AbstractCoordinatePoint{T}}) where {T} = PT(zero(T),zero(T),zero(T))
+CartesianPoint{T}(v::AbstractVector)where {T} = CartesianPoint{T}(v[1], v[2], v[3])
+
+function Base.convert(::Type{CartesianPoint{T}}, pt::CartesianPoint{U}) where {T,U}
+    return CartesianPoint{T}(convert(T, pt.x), convert(T, pt.y), convert(T, pt.z))
+end
+
+
+Base.:(==)(a::CartesianPoint, b::CartesianPoint) = a.x == b.x && a.y == b.y && a.z == b.z
+
+function Base.isapprox(a::CartesianPoint, b::CartesianPoint; kwargs...)
+    return isapprox(a.x, b.x; kwargs...) && isapprox(a.y, b.y; kwargs...) && isapprox(a.z, b.z; kwargs...)
+end
+
+
+@inline Base.:(+)(pt::CartesianPoint, v::CartesianVector) = CartesianPoint(pt.x + v.x, pt.y + v.y, pt.z + v.z)
+@inline Base.:(+)(pt::CartesianPoint, v::CartesianPoint) = CartesianPoint(pt.x + v.x, pt.y + v.y, pt.z + v.z)
```

Comment:
I guess this was added to avoid failing tests, but semantically, this method should not exist then (?)